cps/kni.c

```diff
@@ -668,6 +668,231 @@ rd_modroute(const struct nlmsghdr *req, struct route_update *update)
 	return MNL_CB_OK;
 }
 
+static void
+rd_fill_getroute_reply(struct cps_config *cps_conf, struct nlmsghdr *reply,
+	struct gk_fib *fib, int family, uint32_t seq,
+	uint8_t prefix_len, struct ipaddr **gw_addr)
+{
+	struct rtmsg *rm;
+
+	reply->nlmsg_type = RTM_NEWROUTE;
+	reply->nlmsg_flags = NLM_F_MULTI;
+	reply->nlmsg_seq = seq;
+	reply->nlmsg_pid = cps_conf->nl_pid;
+
+	rm = mnl_nlmsg_put_extra_header(reply, sizeof(*rm));
+	rm->rtm_family = family;
+	rm->rtm_dst_len = prefix_len;
+	rm->rtm_src_len = 0;
+	rm->rtm_tos = 0;
+
```

Comment:
Since the lines above and below are all related to filling the ``rm``, the blank line can be removed here.